app/gui/qt/mainwindow.cpp

```diff
@@ -1587,7 +1625,7 @@ void MainWindow::initPrefsWindow() {
       " yet.<br/>" +
       "We rely on crowdsourcing to help create and maintain translations.<br/>" +
       "<a href=\"https://github.com/samaaron/sonic-pi/blob/master/TRANSLATION.md\">" +
-      "Please consider helping to translate Sonic Pi to your language.</a> "
+      "If you can, please consider helping to translate Sonic Pi to your language.</a> "
```

Comment:
IMO, this is not a polite way to ask for conttibution

Reply:
I didn’t mean to sound bad or impolite, but I see where you’re coming from. I’ll change that back.